docs/DeveloperGuide.md

```diff
@@ -736,13 +874,139 @@ These commands should be tested on the Students Page.
+      Expected: No session is selected. Error details shown in the log box.
+   
+2. Selecting a session while the session list is empty in the Session Page.
+    1. Refer to step 1.2 above under `Selecting a session`
+
+#### Finding a session
+
+1. Finding a session while on the Session Page, with the following sessions - `Tutorial1`, `Tutorial2`, `Lab1` in the list.
+
+    1. Test case: `find session 1`<br>
+       Expected: `Tutorial1` and `Lab1` is displayed. Details of the found sessions shown in the log box.
+
+    1. Test case: `find session tutorial`<br>
+       Expected: `Tutorial1` and `Tutorial2` is displayed. Details of the found sessions shown in the log box.
+
+    1. Test case: `find session Tutorial3` (this session does not exist in the list)<br>
+       Expected: The list is not filtered. Details of no found session(s) shown in the log box.
+
+    1. Other incorrect `find session` commands to try: `find`, `find session`<br>
+       Expected: The list is not filtered. Error details shown in the log box.
+
 [Back to Instructions for Manual Testing](#appendix-instructions-for-manual-testing)
 
 ---
 
 ### Attendance Page Commands
 These commands should be tested on the Attendance Page.
 
+#### Marking selected student(s) as present
+
+1. Marking selected student as present, with at least one student in the list.
+
+   1. Test case: `mark 1`<br>
+      Expected: First student is marked as present. Details of the student marked as present shown in the log box.
+
+   2. Other incorrect `mark` commands to try: `mark`, `mark x` (where x is larger than the student list size)
+      , `mark -1` or `mark 101` (where x is a valid index from the student list) <br>
+      Expected: No student is marked as present. Error details shown in the log box.
+   
+2. Marking selected students as present, with 6 students in the list.
+    1. Test case: `mark 1, 4, 6`<br> 
+        Expected: First, fourth and sixth students are marked as present. Details of the students marked as present shown in the log box.
+    2. Test case: `mark 2, 3, 5, 7` (where there is no student in the seventh index) <br>
+        Expected: No student is marked as present. Error details shown in the log box.
+    3. Other incorrect `mark` commands to try: `mark`, `mark 0` <br>
+       Expected: No student is marked as present. Error details shown in the log box.
+   
+#### Marking selected student(s) as absent
+1. Marking selected student as absent, with at least one student in the list.
+
+    1. Test case: `unmark 1`<br>
+       Expected: First student is marked as absent. Details of the student marked as absent shown in the log box.
+
+    2. Other incorrect `unmark` commands to try: `unmark`, `unmark x` (where x is larger than the student list size)
+       , `unmark -1` or `unmark 101` (where x is a valid index from the student list) <br>
+       Expected: No student is marked as absent. Error details shown in the log box.
```

Comment:
```suggestion
    2. Other incorrect `unmark` commands to try: `unmark`, `unmark x` (where x is larger than the student list size), `unmark -1`<br>
       Expected: No student is marked as absent. Error details shown in the log box.
```

Reason: `unmark 101` is a correct command if index 101 exists for at least one student.